migration/migration_test.go

```diff
@@ -0,0 +1,159 @@
+package migration
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/ipfs/go-datastore"
+	"github.com/ipfs/go-datastore/query"
+	"github.com/stretchr/testify/require"
+	"github.com/textileio/powergate/tests"
+)
+
+func TestEmptyDatastore(t *testing.T) {
+	t.Parallel()
+
+	ms := map[int]Migration{
+		1: func(_ datastore.Txn) error {
+			return fmt.Errorf("This migration shouldn't be run on empty database")
```

Comment:
We define some migration that always fail... so this way we can test that no migrations are run and just the version is set to the lastest value.